src/test/java/ScattTest.java

```diff
@@ -24,13 +25,8 @@
     @Test
     public void testSb2Constructor1() {
         String filePath = Utils.getTestResourcePath("WizardSpells.sb2");
-        try {
-            Sb2 wizardSpells = new Sb2(filePath);
-            assertTrue(wizardSpells.getJSONObject() instanceof org.json.JSONObject);
-        } catch (IOException e) {
-            e.printStackTrace();
-            fail("Constructor threw an error");
-        }
+        Sb2 wizardSpells = new Sb2(filePath);
```

Comment:
I think that this test would still pass if it found a completely different sb2 file and opened that one instead. Does that matter? It would still confirm that it was able to construct, though it might be mishandling the argument. I'm not really sure how I ought to think about writing these tests (whether or not to assume catastrophic problems).